star-wars.Rmd

```diff
@@ -5,16 +5,32 @@ output: github_document
 
 ```{r setup, include = FALSE}
 knitr::opts_chunk$set(echo = FALSE)
+library(dplyr)
+library(tidyr)
+library(knitr)
 ```
 
 ## Most accomplished
 
 Below we look at which 5 characters from Star Wars are the most accomplished pilot (flew the fewest number of starships):
```

Comment:
Shouldn't the most accomplished pilot be the one who flew the **most** ships, not fewest?
```suggestion
Below we look at which 5 characters from Star Wars are the most accomplished pilot (flew the highest number of starships):
```